clang/lib/CodeGen/CGCUDARuntime.h

```diff
@@ -52,7 +52,7 @@ class CGCUDARuntime {
       Texture,  // Builtin texture
     };
 
-    /// The kind flag for an offloading entry.
+    /// The kind bit-field for an offloading entry.
     enum OffloadEntryKindFlag : uint32_t {
```

Comment:
should we move this enum to Utility.h so that we do not duplicate in OfloadWrapper.cpp?

Reply:
That's a good point. I can do that.